lib/ask/runtime/session_mode.ex

```diff
@@ -45,22 +49,22 @@ defmodule Ask.Runtime.SMSSimulatorMode do
 
   defstruct [:channel, :retries]
 
-  def new(channel, _retries), do: %SMSSimulatorMode{channel: channel, retries: []}
+  def new(channel, retries), do: %SMSSimulatorMode{channel: channel, retries: retries}
```

Comment:
Do we care about retries in Simulator?

Reply:
No, but as the data was there it seemed to me unnecessary to ignore it